WooCommerce/Classes/POS/Utils/Color+WooCommercePOS.swift

```diff
@@ -28,6 +28,10 @@ extension Color {
         Color(uiColor: .wooCommercePurple(.shade60))
     }
 
+    static var wooPurple10: Color {
+        Color(red: 207.0 / 255.0, green: 185.0 / 255.0, blue: 246 / 255.0)
```

Comment:
`Color(uiColor: .wooCommercePurple(.shade10))`
We have these woo purple shades in `ColorPalette` assets

Reply:
Thanks, didn't notice that, fixed!